src/extension.ts

```diff
@@ -237,7 +238,9 @@ export async function activate(context: vscode.ExtensionContext) {
 
         recordToolkitInitialization(activationStartedOn, getLogger())
 
-        globals.telemetry.assertPassiveTelemetry(globals.didReload)
+        if (!isReleaseVersion()) {
```

Comment:
No need to show the error message in a prod version. Expanding the CDK nodes before full activation caused this to happen.